website/src/components/explorer/map.jsx

```diff
@@ -205,11 +219,14 @@ export function ExplorerMap({
       ref={deckRef}
       layers={layers}
       initialViewState={currentInitialViewState}
-      controller={true}
       getTooltip={getTooltip}
       getCursor={getCursor}
       onClick={onClick}
       onLoad={() => setDeckLoaded(true)}
+      controller={{
+        dragPan: windowWidth && windowWidth >= MOBILE_CUTOFF_WINDOW_WIDTH,
+      }}
+      touchAction="pan-y"
```

Comment:
What do these two properties alter, specifically?

Reply:
They're suggested by the deck.gl docs: https://deck.gl/docs/api-reference/core/deck#touchaction

* Disable drag to pan when the window width is too low. Single finger touch interactions should not trigger moving the map.
* I think what touchAction does is allow that touch interaction to go to the page rather than get consumed by deck.gl.